websites/K/KQM/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "290820916051705868",
+		"name": "clevernt"
+	},
+	"service": "KQM",
+	"description": {
+		"en": "KQM is home to the best Genshin Impact guides and the most reputable theorycrafting community. Learn about your favorite Genshin Impact characters’ builds, artifacts, weapons, teams, and more!"
+	},
+	"url": "keqingmains.com",
+	"matches": ["*://*.keqingmains.com/*"],
+	"iFrameMatches": ["*://*.keqingmains.com/*"],
```

Comment:
iframe matches is not needed since you dont have any iframes that you inject into.